cvat-canvas/src/typescript/masksHandler.ts

```diff
@@ -627,6 +627,8 @@ export class MasksHandlerImpl implements MasksHandler {
                     enabled: true,
                     shapeType: 'mask',
                 };
+
+                this.onDrawRepeat({ enabled: true, shapeType: 'mask' });
```

Comment:
Why do we call it twice? isnt second call already contains data with enabled and mask shape type?

Reply:
In case with masks no 
For masks first call initiates drawing and second is necessary to setup brush tools